react/DynamicIframe.tsx

```diff
@@ -14,7 +14,14 @@ const DynamicIframe: StorefrontFunctionComponent<DynamicIframeProps> = ({
 
   const src = dynamicSrc.replace(/({[A-z0-9]*})/g, function(match: string) {
     const thisParam = match.replace(/{|}/g, '')
-    return params[thisParam]
+    if (thisParam && params[thisParam]) {
+      return params[thisParam]
+    } else {
```

Comment:
Remove `else`, because it's not needed since we are already returning inside the `if`